File: tests/python/in/900.c
int z =4;
int factorial(int a) {
    if(a==1){
        return 1;
    }
    else{
        return a*factorial(a-1) ;
    }
}

int x=3;

int main(){   
    return factorial(5)+factorial(x)+factorial(z);
}

File: tests/python/in/900.py
z=4

def factorial(a) :
    global z
    if(a==1):
        return 1
    else:
        return a*factorial(a-1)

x=3

def main():
    global z
    global x   
    return factorial(5)+factorial(x)+factorial(z)

# Boilerplate
if __name__ == "__main__":
    import sys
    ret=main()
    sys.exit(ret)

